api/src/test/java/uk/gov/dhsc/htbhf/claimant/message/MessagePayloadFactoryTest.java

```diff
@@ -73,4 +73,31 @@ void shouldBuildSendNewCardSuccessEmailPayloadWithOnlyPregnancyPayment() {
         assertEmailPayloadCorrectForClaimantWithPregnancyVouchersOnly(payload.getEmailPersonalisation(), endDate);
     }
 
+    @Test
+    void shouldBuildPaymentNotificationEmailPayloadWithAllPaymentTypes() {
+
+        LocalDate startDate = LocalDate.now();
+        LocalDate endDate = startDate.plusDays(28);
+        PaymentCycle paymentCycle = aPaymentCycleWithStartAndEndDate(startDate, endDate);
+
+        EmailMessagePayload payload = MessagePayloadFactory.buildPaymentNotificationEmailPayload(paymentCycle);
+
+        assertThat(payload.getClaimId()).isEqualTo(paymentCycle.getClaim().getId());
+        assertThat(payload.getEmailType()).isEqualTo(EmailType.PAYMENT);
+        assertEmailPayloadCorrectForClaimantWithAllVouchers(payload.getEmailPersonalisation(), endDate);
```

Comment:
This method doesn't check that the field `payment_amount` is present in the Map so that should be updated too.